frontend/rust-lib/flowy-grid/src/event_map.rs

```diff
@@ -42,63 +42,136 @@ pub fn create(grid_manager: Arc<GridManager>) -> Module {
     module
 }
 
+
+/// [GridEvent] defines events that are used to interact with the Grid. You could check [this](https://appflowy.gitbook.io/docs/essential-documentation/contribute-to-appflowy/architecture/backend/protobuf)
+/// out, it includes how to use these annotations: input, output, etc.
 #[derive(Clone, Copy, PartialEq, Eq, Debug, Display, Hash, ProtoBuf_Enum, Flowy_Event)]
 #[event_err = "FlowyError"]
 pub enum GridEvent {
+    /// [GetGrid] event is used to get the [GridPB]
+    ///
+    /// The event handler accepts [GridIdPB] and return [GridPB] if there is no errors.
+    ///
     #[event(input = "GridIdPB", output = "GridPB")]
     GetGrid = 0,
 
+    /// [GetGridBlocks] event is used to get the grid's block.
+    ///
+    /// The event handler accepts [QueryGridBlocksPayloadPB] and return [RepeatedGridBlockPB]
+    /// if there is no errors.
     #[event(input = "QueryGridBlocksPayloadPB", output = "RepeatedGridBlockPB")]
     GetGridBlocks = 1,
 
+    /// [GetGridSetting] event is used to get the grid's setting.
+    ///
+    /// The event handler accepts [GridIdPB] and return [GridSettingPB]
+    /// if there is no errors.
     #[event(input = "GridIdPB", output = "GridSettingPB")]
```

Comment:
Settings is usually plural.  There is not one setting, there are many.  We should try to fix this as we go along.

Reply:
GridSettingPB contains lots of settings. We could extend the GridSettingPB.